dash/development/_py_components_generation.py

```diff
@@ -570,6 +570,10 @@ def array_of():
             )
         return "list"
 
+    def tuple_of():
+        elements = [js_to_py_type(element) for element in type_object["elements"]]
+        return f"tuple of ({', '.join(elements)})"
```

Comment:
The word `tuple` has a different meaning in Python than it has in TS, and it'll serialize as a list, so perhaps something like this?

```suggestion
        return f"list of {len(elements)} elements: [{', '.join(elements)}]"
```